packages/editor-ui/src/api/mfa.ts

```diff
@@ -24,6 +24,7 @@ export async function verifyMfaToken(
 
 export type DisableMfaParams = {
 	token: string;
+	recoveryCode: string;
 };
```

Comment:
I always send both because in the backend we have a function that check both and return true/false depending on whether one of the them is valid. It's the same function to login with MFA. The alternative would have been to check in the FE, and in the BE for which one we are sending and validate always only the one we send, and that would have been some IF's conditions that I wanted to avoid.

Reply:
But yeah, I see where you are coming from that is kind of consuming that you need to send both. Will address